### PR TITLE
Add more legends to the HOF

### PR DESCRIPTION
**Pull request by 0tu**
Adding more users who have leetcoded their way to a new role to the HOF. 

I have never used Go before so it's a coin toss on whether or not my formatting changes will work but we can test that in prod of course.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -3,13 +3,21 @@ package main
...
 	"os"
+    "fmt"
 
 	widget "github.com/ketchupsalt/slack-widget"
 
 	"github.com/slack-go/slack"
 	"github.com/slack-go/slack/slackevents"
 )
 
+var hofer_str = `
```
Comment (smortime) [inline main.go]: please make this a constant

File: main.go [hunk 2]
```diff
@@ -3,13 +3,21 @@ package main
 import (
```
Comment (smortime) [inline main.go:3]: please run `go fmt .`

I should make a githook cause I forget to do this too :zany_face:

Comment (0tu) [reply main.go:3]: Oh, so you're suggesting I actually download Go on my computer now?! 😞

**Review by smortime — APPROVED** [review]
LGTM :palm_tree:

**Pull request merged**